packages/cfd/src/Helpers/compare-accounts-config.ts

```diff
@@ -41,8 +41,226 @@ const getHighlightedIconLabel = (trading_platforms: TTradingPlatformAvailableAcc
+            return {
+                ...cfdConfig,
+                counterpartyCompany: 'Deriv (V) Ltd',
+                jurisdiction: 'Vanuatu',
+                jurisdictionDescription: 'Jurisdiction',
+                regulator: 'Vanuatu Financial Services Commission',
+                regulatorDescription: 'Regulator/External dispute resolution',
+            };
+        case 'financial_svg':
+            return {
+                ...cfdConfig,
+                counterpartyCompany: 'Deriv (SVG) LLC',
+                jurisdiction: 'St. Vincent & Grenadines',
+                jurisdictionDescription: 'Jurisdiction',
+                regulator: 'Financial Commission',
+                regulatorDescription: 'Regulator/External dispute resolution',
+            };
+        case 'financial_bvi':
+            return {
+                ...cfdConfig,
+                counterpartyCompany: 'Deriv (BVI) Ltd',
+                jurisdiction: 'British Virgin Islands',
+                jurisdictionDescription: 'Jurisdiction',
+                regulator: 'British Virgin Islands Financial Services Commission',
+                regulatorDescription: '(License no. SIBA/L/18/1114) Regulator/External Dispute Resolution',
+            };
+        case 'financial_vanuatu':
+            return {
+                ...cfdConfig,
+                counterpartyCompany: 'Deriv (V) Ltd',
+                jurisdiction: 'Vanuatu',
+                jurisdictionDescription: 'Jurisdiction',
+                regulator: 'Vanuatu Financial Services Commission',
+                regulatorDescription: 'Regulator/External Dispute Resolution',
+            };
+        case 'financial_labuan':
+            return {
+                ...cfdConfig,
+                leverage: '1:100',
+                counterpartyCompany: 'Deriv (FX) Ltd',
+                jurisdiction: 'Labuan',
+                jurisdictionDescription: 'Jurisdiction',
+                regulator: 'Labuan Financial Services Authority',
+                regulatorDescription: '(licence no. MB/18/0024) Regulator/External Dispute Resolution',
+            };
+        // Dxtrade
+        case 'all_':
+        case 'all_svg':
+            return {
+                ...cfdConfig,
+                counterpartyCompany: 'Deriv (SVG) LLC',
+                jurisdiction: 'St. Vincent & Grenadines',
+                jurisdictionDescription: 'Jurisdiction',
+                regulator: 'Financial Commission',
+                regulatorDescription: 'Regulator/External dispute resolution',
+            };
+        default:
+            return cfdConfig;
+    }
+};
+
+// Sort the MT5 accounts in the order of derived, financial and swap-free
+const getSortedAvailableAccounts = (available_accounts: TModifiedTradingPlatformAvailableAccount[]) => {
+    const swap_free_accounts = available_accounts
+        .filter(item => item.market_type === 'all')
+        .map(item => ({ ...item, platform: 'mt5' } as const));
```

Comment:
There's no need to use map here. Removing it would give you the same result.

Reply:
I am trying to add the platform property inside all the jurisdiction objects. Thats why i am using map without it the platform property will not be added to the corresponding objects.